submissions/GeometricalMLforNeuroscience.md

```diff
@@ -0,0 +1,215 @@
+  
+The progress made using tools from statistical physics to the understanding of network performances has grown recently grown faster. At its core techniques lie the use of tools from random matrix theory (the study of matrix where each element comes from a certain random distribution) and result from probability theory. A recent paper providing new results as well as a good summary of the literature was recently released and focus on an analytical expression of the loss of teacher-student models. [31]
+ 
+More generally, the use of random matrix theory is instrumental in neuroscience. For example, it allows to have an estimate of the number of principal components that accounts for true feature of the data rather than noise. [32]
+
+The theoretical work on learning is vast, we will therefore restrict our analysis of the literature by considering cases were theoretical tools make use of the geometrical properties of the measured neuronal data. In doing so we would like to highlight how differential geometry may serve as an important toolbox for theorist and for experimentalists.
+
+
+# Differential Geometry: theoretical tools
+
+Differential geometry gathers a set of mathematical tools to study manifolds: how can we define maps from one manifold to another? How can we study the propagation of a wave on a manifold? This toolbox was extensively used to formulate physical theory.  In neuroscience, its importance was stressed as it helped to provide an understanding of the nature of visual hallucinations.  [33]
+ 
+More recently, such tools were used to justify theoretically that computations of deep neural network could transform a complex, ie curved, manifolds (inputs) into a simpler, flat, one [34]
+ 
+In fact, the author observed that when the dynamics approach a state of chaos (where tiny variation in initial condition leads to diverging trajectories), the network could become exponentially expressive. By setting its dynamics near the transition to chaos, the network may compute varied functions. This phenomenon was further elucidated in subsequent works. More recently, a clearer picture of chaotic RNN was obtained [35]. The author discovered the Lyapunov spectra of such network, this spectrum measures the rate of change of diverging trajectory. It can also be used to discover the dimensionality of the attractor on which the dynamic remains constrained.
+  
+Such attractor are also becoming more and more characterized. Differential geometry is at the hearth of the analysis of dynamical systems. In this manner, it can be used to gather a simpler understanding of the computations recurrent neural network perform. In [36], it was observed that the dynamics of various network trained to classify text could be interpreted as evidence accumulation on a low-dimensional manifold, depending on the input dataset. Similarly, the proof that an attractor exists in experimentally measured neuronal dynamics was made studying the head-direction ring. [37]
```

Comment:
How are attractors related to neural computations?

Reply:
They allow to fight (part) of the noise emerging from either the inputs sensors or from the local noisy computations. They are also seen as the basis of some working memory model! Will update the text to make it more explicit.